src/Product.php

```diff
@@ -367,6 +367,45 @@ public function setShippingWeight($str)
         return $this;
     }
 
+    /**
+     * Add shipping length
+     * 
+     * @param string $str
+     *
```

Comment:
I think it's nice to have a link to a documentation about this field. Something like this:

`@see https://support.google.com/merchants/answer/6324498?hl=en`

And in other places too.